identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeProofingState.kt

```diff
@@ -69,50 +73,66 @@ class FunkeProofingState(
     suspend fun getEvidenceRequests(env: FlowEnvironment): List<EvidenceRequest> {
         return if (access == null) {
             if (!tosAcknowleged) {
-                val message = env.getInterface(Resources::class)!!
-                    .getStringResource("funke/tos.html")!!
-                listOf(EvidenceRequestMessage(
-                    message = message,
-                    assets = emptyMap(),
-                    acceptButtonText = "Continue",
-                    rejectButtonText = "Cancel"
-                ))
+                val message = if (useGermanEId) {
+                    env.getInterface(Resources::class)!!
+                        .getStringResource("funke/tos.html")!!
+                } else {
+                    val metadata = Openid4VciIssuerMetadata.get(env, credentialIssuerUri)
+                    val issuingAuthorityName = metadata.display[0].text
+                    val documentName = metadata.credentialConfigurations[credentialConfigurationId]!!.display[0].text
+                    env.getInterface(Resources::class)!!
+                        .getStringResource("generic/tos.html")!!
+                        .replace("\$ISSUER_NAME", issuingAuthorityName)
+                        .replace("\$ID_NAME", documentName)
+                }
+                listOf(
+                    EvidenceRequestMessage(
+                        message = message,
+                        assets = emptyMap(),
+                        acceptButtonText = "Continue",
+                        rejectButtonText = "Cancel"
+                    )
+                )
             } else if (!notificationPermissonRequested) {
-                listOf(EvidenceRequestNotificationPermission(
-                    permissionNotGrantedMessage = """
-                        ## Receive notifications?
-                        
-                        If there are updates to your document the issuer will send an updated document
-                        to your device. If you are interested, we can send a notification to make you aware
-                        of when this happens. This requires granting a permission.
-                        
-                        If you previously denied this permission, attempting to grant it again might not do
-                        anything and you may need to go into Settings and manually enable
-                        notifications for this application.
-                    """.trimIndent(),
-                    grantPermissionButtonText = "Grant Permission",
-                    continueWithoutPermissionButtonText = "No Thanks",
-                    assets = mapOf()
-                ))
+                listOf(
+                    EvidenceRequestNotificationPermission(
```

Comment:
Is this just a formatting change? I don't see any code changes to this block.
And was this auto-formatted, or did you update the formatting?

Reply:
formatted it back